src/index.jsx

```diff
@@ -24,6 +26,21 @@ import NotificationCourses from './notification-preferences/NotificationCourses'
 import NotificationPreferences from './notification-preferences/NotificationPreferences';
 
 subscribe(APP_READY, () => {
+  datadogRum.init({
+    applicationId: 'a3f99dcb-4955-4baa-8341-39a88603ab08',
+    clientToken: 'pubf2e79d946cec4c4413965620ba0e0b72',
+    site: 'datadoghq.com',
+    service: 'edx-frontend-sandbox',
+    env: 'staging',
+    // Specify a version number to identify the deployed version of your application in Datadog
+    version: '1.0.0',
+    sessionSampleRate: 100,
+    sessionReplaySampleRate: 20,
```

Comment:
@abdullahwaheed @schenedx @jristau1984: [inform] 
1. We are likely going to need to tune `sessionSampleRate` during the migration to keep within the contract. I'm guessing closer to 50%, but we'll need to see. We might tune some like prospectus even lower?
2. We are on an older contract that allows us to get some session replays, which is an interesting feature we didn't have with NR and may help us and support. More to look into when you get there.